README.md

```diff
@@ -1,9 +1,106 @@
-# python-symbiflow-v2x
+# SymbiFlow Verilog to XML
```

Comment:
I think we want to be clear it is "Verilog to Routing XML", however "SymbiFlow Verilog to Verilog to Routing XML" is super weird looking with multiple "Verilog to" in it....

Maybe the following? Open to alternatives;
```suggestion
# Verilog to Routing XML file generation from Verilog (`python-symbiflow-v2x`)
```

Reply:
Poke.

Reply:
Alright changed. I thought I changed this, maybe lost it when rebasing.

Reply:
Still doesn't seem to have updated?

Reply:
Oh! The issue is you have a `README.rst` and a `README.md` file now!

Reply:
Oh I forgot about it. I changed to use README.rst in the other pull request.